package.json

```diff
@@ -15,11 +15,14 @@
   },
   "dependencies": {
     "@phosphor-icons/react": "^2.1.7",
+    "@radix-ui/react-tooltip": "^1.1.8",
```

Comment:
curious to know the use case, we would love to not add more dependencies to the project @VarshiniSuresh90

Reply:
incase you want to use a tooltip, we have a component for tooltip that we have built. You can use that